src/main/java/org/cascadebot/cascadebot/utils/ConfirmUtils.java

```diff
@@ -60,6 +60,10 @@ public static boolean confirmAction(long userId, String actionKey, TextChannel c
                     if (runner.getIdLong() != action.userId) return;
                     action.run();
                 }));
+                group.addButton(new Button.UnicodeButton(UnicodeConstants.RED_CROSS, ((runner, channel1, message1) -> {
```

Comment:
should this be a parameter in case we want a confirm that gets timed out?
@mrjvs what do you think?